src/pages/AdvancedOptionsDoc.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="container">
+    <h1>
+      Advanced Options Documentation
+    </h1>
+    <div class="text-start mx-5 mt-8">
+      <h2 class="mb-3">
+        Multi-GPU Options
+      </h2>
+      <v-expansion-panels variant="accordion">
+        <v-expansion-panel>
+          <v-expansion-panel-title>
+            <h3 class="expansion-panel-title">
+              gpu-id
+            </h3>
+          </v-expansion-panel-title>
+          <v-expansion-panel-text>
+            <code>
+              <p>gpu device to use (default=auto) can be 0,1,2 for multi-gpu.</p>
+            </code>
+          </v-expansion-panel-text>
+        </v-expansion-panel>
+        <v-expansion-panel>
+          <v-expansion-panel-title>
+            <h3 class="expansion-panel-title">
+              tile-size
+            </h3>
+          </v-expansion-panel-title>
+          <v-expansion-panel-text>
+            <code>
+              <p>Gpu device to use (default=auto) can be 0,1,2 for multi-gpu.</p>
```

Comment:
This is for the gpu-id, not tile-size.

```suggestion
              <p>Tile size (>=32/0=auto, default=0) can be 0,0,0 for multi-gpu.</p>
```